docs/source/upcoming_release_notes/1286-adding_more_targets_into_pa1k4.rst

```diff
@@ -0,0 +1,30 @@
+1286 adding more targets into pa1k4
+#################
+
+API Breaks
+----------
+- N/A
+
+Library Features
+----------------
+- N/A
+
+Device Features
+---------------
+- Adding two more targets in device
```

Comment:
```suggestion
- Changes `CalibrationAxis` from 5 -> 7 states
```

Reply:
I just changed and push new doc. Thanks!